bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -155,4 +172,12 @@ public CloudBigtableScanConfiguration(String projectId, String zone, String clus
   public Scan getScan() {
     return serializableScan.scan;
   }
+
+  @Override
+  public Builder toBuilder() {
```

Comment:
Different Builder classes, I see.
.setConfiguration(configuration) looks like constructor to me then.
but I am OK with what you have.